libs/core/models/anime.ts

```diff
@@ -6,23 +6,23 @@ import { ProductionStatuses } from './production-statuses';
 export interface Anime {
 
 	/** ID. */
-	id: number;
+	readonly id: number;
 
-	/** Title. */
-	titleEng: string;
+	/** Title (ENG). */
+	readonly titleEng: string;
 
 	/** Image. */
-	image: string;
+	readonly image: string;
```

Comment:
Please provide some details about this image string. Is a link or 'base64' string of the image, or maybe it's something else?
For example you can rename it like this:
```suggestion
	readonly thumbnailUrl: string;
```